tests/Crypto/tst_cryptorequests/tst_cryptorequests.cpp

```diff
@@ -109,42 +114,60 @@ private slots:
 private:
     void addCryptoTestData()
     {
+        QTest::addColumn<CryptoManager::Algorithm>("algorithm");
```

Comment:
Note that cipherEncryptDecrypt_data() needs to have the CryptoManager::AlgorithmAes test parameter added to the test data rows as well.

Reply:
Will fix, thanks!